packages/api-cardano-db-hasura/test/util.ts

```diff
@@ -11,19 +11,27 @@ import util from '@cardano-graphql/util'
 // import { Genesis } from '@src/graphql_types'
 // import { CardanoNodeClient } from '@src/CardanoNodeClient'
 
-const getLastConfiguredMajorVersion = (network: string) =>
-  require(`../../../config/network/${network}/cardano-node/config.json`)['LastKnownBlockVersion-Major']
+// const getLastConfiguredMajorVersion = (network: string) =>
```

Comment:
These arguments are are not required for the e2e client. I will tidy up in a separate commit prior to the production release